src/gui/addnewtorrentdialog.cpp

```diff
@@ -214,7 +214,11 @@ void AddNewTorrentDialog::show(QString source, const BitTorrent::AddTorrentParam
             ok = dlg->loadTorrent(source);
 
         if (ok)
+#ifdef Q_OS_MAC
```

Comment:
Good idea, but definitely out of the scope of this PR, especially because I do not have the dev environment set for mac. I could imagine someone removing the ifdefs for both non-modal dialogues and subcategory padding =)